CHANGELOG.md

```diff
@@ -1,6 +1,44 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project will be documented in this file
+
+## v20.11.0
+
+### Added
+
+* Activate SAML when using single sign-on (SSO)
+* Add a route in OpenShift to allow connections to SSO, when enabled
+* Updated the `ServerEAR.Dockerfile` to reduce layers
+* Added note with fix needed for an update in IBM MQ, the details of which can be found
+[here](https://github.com/IBM/charts/blob/master/stable/ibm-mqadvanced-server-dev/RELEASENOTES.md)
+* Added links to Architecture and Troubleshooting sections from within the flow of the document
+
+### Changed
+
+* Updated SPM 7.0.10.0 Supported Prereqs.
+  * Kubernetes version 1.18 support introduced.
+  * Kubernetes version 1.16 is now in a state of deprecated.
+* Clarified MQ configuration reference for container or VM.
+* Moved IKS and OpenShift considerations into Architecture
+* Updated Nav items for considerations. *Note:* any bookmarked pages will no longer work
+* Replaced inline links with anchor links in "MustGather"
+* Provided rationale for building own images
+* Provided clarification on use of `$PROJECT` environment variable
+* Link to helm defect for the need to repush docker images
+* Provided clarification on the use of helm releasename
+* Provided clarification on the use of subnet
+* Restructure command examples so that required additional information is before the command
+* Update dev workstation archtitecture diagram to differentiate the cluster content
```

Comment:
Typo: ~~archtitecture~~ _architecture_

Also, consider rephrasing to: _Update architecture diagram to differentiate the cluster content for a local development workstation_

Reply:
👍